assets/js/admin/admin.js

```diff
@@ -221,4 +221,61 @@
 		$(this).append(video);
 
 	});
+
+	// Toggles edit state.
+	function toggleEditTitle( e ) {
+		e.preventDefault();
+
+		var $ele = $( '#edit-form-name' ),
+			$input_title = $ele.siblings( '#evf-edit-form-name' );
+		// Toggle disabled property.
+		$input_title.prop( 'disabled' , function(_, val) {
+			return ! val;
+		});
```

Comment:
A blank line after any statements as for coding standard.

Reply:
The spacing now abides coding standards.